OpenRA.Mods.Common/Scripting/Properties/PlayerProperties.cs

```diff
@@ -39,18 +39,25 @@ public string Race
 			get
 			{
 				Game.Debug("The property `PlayerProperties.Race` is deprecated! Use `PlayerProperties.Faction` instead!");
-				return Player.PlayerReference.Faction;
+				return Player.Faction.InternalName;
 			}
 		}
 
 		[Desc("The player's faction.")]
-		public string Faction { get { return Player.PlayerReference.Faction; } }
+		public string Faction { get { return Player.Faction.InternalName; } }
```

Comment:
I chose the internal name here because that's what's in `PlayerReference.Faction`. However, for scripting/display purposes, it might be nicer to have the real faction's name. Do we want to break backward compatibility here? Add a second property?

Currently, this property isn't used at all, at least by our shipped maps.

//cc @abcdefg30